Classes/Driver/AbstractQuery.php

```diff
@@ -119,7 +135,6 @@ public function aggregation($name, array $aggregationDefinition, $parentPath = '
      * @param string $parentPath The parent path to add the sub aggregation to
      * @param string $name The name to identify the resulting aggregation
      * @param array $aggregationConfiguration
-     * @return QueryInterface
```

Comment:
So it should be `@ return void`now?

Reply:
For those little nitpick, I think we can use the github edit feature, add the missing bit and done ;)